src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.GssApiException.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurityNative
+    {
+        internal sealed class GssApiException : Exception
+        {
+            private Status _minorStatus;
+
+            public Status MinorStatus
+            {
+                get { return _minorStatus;  }
+            }
+
+            public GssApiException(string message) : base(message)
+            {
+            }
+
+            public GssApiException(Status majorStatus, Status minorStatus)
+                : this(SR.Format(SR.net_gssapi_operation_failed, majorStatus, minorStatus), majorStatus, minorStatus)
+            {
+            }
+
+            public GssApiException(string message, Status majorStatus, Status minorStatus)
```

Comment:
This ctor seems like it's only used by the previous ctor, and if it is used by other code, we'd end up not including majorStatus and minorStatus in the error message.  Seems like it'd be better to do away with this ctor entirely and just have the previous one be:
```C#
public GssApiException(Status majorStatus, Status minorStatus) :
    base(SR.Format(SR.net_gssapi_operation_failed, majorstatus, minorStatus))
{
    HResult = (int)majorStatus;
    _minorStatus = minorStatus;
}
```